plugins/janus_videoroom.c

```diff
@@ -3005,6 +3008,7 @@ static int janus_videoroom_access_room(json_t *root, gboolean check_modify, gboo
 		}
 	}
 	if(check_join) {
+		JANUS_LOG(LOG_INFO, "check_join = true (%s)\n", room_id_str);
```

Comment:
This looks like a debug line that was left there?

Reply:
removed